OracleAccessManagement/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,401 @@
+c) Run the following commands to setup imagetool:
+
+```
+$ cd <work directory>/imagetool/bin
+$ source setup.sh
+```
+
+d) Execute the following to validate the WebLogic Image Tool:
+
+```
+$ ./imagetool.sh --version
+imagetool:1.9.3
+```
+
+On pressing tab after typing `imagetool` on the command line, it will display the subcommands available in the imagetool:
+
+```
+$ ./imagetool.sh <TAB>
+cache   create  help    rebase  update
+```
+
+e) The Image Tool creates a temporary Docker context directory, prefixed by wlsimgbuilder_temp, every time the tool runs. Under normal circumstances, this context directory will be deleted. However, if the process is aborted or the tool is unable to remove the directory, it is safe for you to delete it manually. By default, the Image Tool creates the Docker context directory under the user's home directory. If you prefer to use a different directory for the temporary context, set the environment variable `WLSIMG_BLDDIR`.
+
+```
+$ export WLSIMG_BLDDIR="/path/to/dir"
+```
+
+f) The Image Tool maintains a local file cache store. This store is used to look up where the Java, WebLogic Server installers, and WebLogic Server patches reside in the local file system. By default, the cache store is located in the user's $HOME/cache directory. Under this directory, the lookup information is stored in the .metadata file. All automatically downloaded patches also reside in this directory. You can change the default cache store location by setting the environment variable `WLSIMG_CACHEDIR`.
+
+```
+$ export WLSIMG_CACHEDIR="/path/to/cachedir"
+```
+
+
+
+# 4. Download the required packages/installers&Patches
+
+a) Download the required installers from the [Oracle Software Delivery Cloud](https://edelivery.oracle.com/) and save them in a directory of your choice e.g: \<work directory\>/stage:
+
+* Oracle Identity and Access Management 12.2.1.4.0
+* Oracle Fusion Middleware 12c Infrastructure 12.2.1.4.0
+* Oracle JDK 
+
+
+**Note** : the required list of packages/installers & patches for specific bundled patchsets can be found in the latest manifest file. For example, the list below displays the packages/installers & patches from manifest.oam.july2020.properties:
+
+```
+[JDK]
+jdk-8u261-linux-x64.tar.gz
+
+[INFRA]
+fmw_12.2.1.4.0_infrastructure.jar
+
+[INFRA_PATCH]
+p28186730_139424_Generic.zip:Opatch
+p31537019_122140_Generic.zip:WLS
+p31544353_122140_Linux-x86-64.zip:WLS
+p31470730_122140_Generic.zip:COH
+p31488215_122140_Generic.zip:JDEV
+
+
+[OAM]
+fmw_12.2.1.4.0_idm.jar
+
+[OAM_PATCH]
+p31556630_122140_Generic.zip:OAM
+```
+
+b) Download any patches listed in the manifest file from [My Oracle Support](https://support.oracle.com) and copy to \<work directory\>/stage.
+
+
+
+# 5. Required build files
+
+
+a) The OAM image requires additional files for creating the OAM domain and starting the WebLogic Servers. Download the required files from the FMW [repository](https://github.com/oracle/docker-images/). For example:
```

Comment:
This is not the FMW repository, nor do you download files from it. Just say, "Sync this repo to your local machine"